packages/shared/components/BalanceSummarySection.svelte

```diff
@@ -32,16 +37,28 @@
         }
     }
 
-    function handleAmount(isBaseToken: boolean, amount: number) {
-        return isBaseToken ? getAmount(amount) : getAmountMana(amount)
+    function handleAmount(isBaseToken: boolean = false, amount: number) {
```

Comment:
This should be receiving `asset` as function parameter, just like `isBaseToken` and `amount`